src/helpers/helpers.options.js

```diff
@@ -143,3 +148,51 @@ export function resolve(inputs, context, index, info) {
 		}
 	}
 }
+
+const ucFirst = (str) => str.charAt(0).toUpperCase() + str.slice(1);
+
+const notIndexable = ['borderDash', 'fill'];
+const indexable = (key, index) => notIndexable.indexOf(key) !== -1 ? undefined : index;
+
+/**
+ * @param {*} elementType
```

Comment:
This function is used enough in core code that it could be helpful to add descriptions to these parameters